templates/projects/tasks_browse.webpack.ejs

```diff
@@ -652,15 +654,17 @@
 </div>
 
 <!-- Project owner, subadmin that are project co-owners and admins can edit/delete task properties -->
```

Comment:
The comment will also need to be updated accordingly.

Reply:
done. Thanks!